skrf/__init__.py

```diff
@@ -5,7 +5,7 @@
 # Python 3 compatibility
 from __future__ import absolute_import, print_function, division
 
-__version__ = '0.23.1'
+__version__ = '0.23.2dbanas'
```

Comment:
Please remove this change of version number.

Reply:
**==> Changed as requested in next push.**